app/destination/page.js

```diff
@@ -75,40 +97,46 @@ export const Destinations = () => {
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
-          <div className={styles.planetCard}>
-            <img className={styles.planetThumbnail} src="/destination/image-europa.png" alt="" />
-            <div className={styles.planetDescription}>
-              <h2>EUROPA {isPlanetSelected ? "- SELECTED" : ""}</h2>
-              <p>Lorem ipsum...</p>
-            </div>
-            <button 
-              className="roundButton" 
-              onClick={() => onAddOrRemovePlanet('Pluto', 0)}
-            > 
-              {isPlanetSelected ? "REMOVE" : "ADD PLANET"}
-            </button>
-          </div>
+
+          <PlanetCard
+            name="Europa"
+            description="Europa, one of Jupiter’s moons, is an icy world with a hidden ocean beneath its surface. This mysterious moon is a prime candidate for the search for extraterrestrial life, making it a thrilling destination for space explorers."
+            thumbnail="/destination/image-europa.png"
+            isSelected={selectedPlanets.includes("Europa")}
+            onAddOrRemovePlanet={() => onAddOrRemovePlanet("Europa")}
+          />
+
+          <PlanetCard
+            name="Moon"
+            description="Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers."
+            thumbnail="/destination/image-moon.png"
+            isSelected={selectedPlanets.includes("Moon")}
+            onAddOrRemovePlanet={() => onAddOrRemovePlanet("Moon")}
+          />
+
+          <PlanetCard
+            name="Mars"
+            description="Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth."
+            thumbnail="/destination/image-mars.png"
+            isSelected={selectedPlanets.includes("Mars")}
+            onAddOrRemovePlanet={() => onAddOrRemovePlanet("Mars")}
+          />
+
+          <PlanetCard
+            name="TITAN"
+            description="Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets."
+            thumbnail="/destination/image-titan.png"
+            isSelected={selectedPlanets.includes("TITAN")}
+            onAddOrRemovePlanet={() => onAddOrRemovePlanet("TITAN")}
+          />
```

Comment:
Good job using the component here!  You could save the information in an array and then map through it to make it more modular!